MatiasDure/Project_IMTS
# Burble storm passive

**Description**
Passive event where the bee get suck into the storm 

**Showcase**
![bee storm](https://github.com/user-attachments/assets/be3f3646-1a2e-4925-8d2f-80c65095067c)

**How to test**

- Open the BurbbleStorm scene
- Play the scene
- Check the bee get suck to storm
- Tweak the storm spec for modification
- Replay the scene
